src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -222,6 +222,8 @@ function getCSharpInputTypeKindByIntrinsicModelName(
                     );
                     return InputPrimitiveTypeKind.DurationISO8601;
             }
+        case "azureLocation":
```

Comment:
Just to confirm, compiler would mark the `azureLocation` with kind `azureLocation`?

Reply:
its not kind, its Intrinsic name. azureLocation is a scalar.